src/views/InvolvementProfile/involvement-profile.scss

```diff
@@ -7,3 +7,15 @@
     cursor: pointer;
   }
 }
+
+.member-list-image {
+  border-radius: 0.5rem;
+  height: 70px;
+  width: 70px;
+}
+
+.contact-list-image {
+  border-radius: 0.5rem;
+  height: 70px;
+  width: 70px;
+}
```

Comment:
These two classes are never used, so they should be removed.

Reply:
Done!!!